### PR TITLE
Add patch for duplicates tracking

### PR DESCRIPTION
**Pull request by squirrelo**
This adds a table for tracking duplicate survey participant names and populates it.

### CODE REVIEW

File: amgut/db/patches/0029.sql [hunk 1]
```diff
@@ -0,0 +1,174 @@
+-- Oct 27, 2015
+-- Add way to track what surveys are duplicates for participant consent
+
+--Add timestamp for when a kit is registered
+ALTER TABLE ag.ag_kit ADD registered_on timestamp DEFAULT current_timestamp;
+
+--Add table to track duplicates
+CREATE TABLE ag.duplicate_consents (
```
Comment (wasade) [inline amgut/db/patches/0029.sql]: no explicit pk? the indexing scheme below suggests either having a `unique` constraint over all three columns, or to create an aggregate natural primary key of all three columns. what do you think?

Comment (squirrelo) [inline amgut/db/patches/0029.sql]: agreagate PK would work best, with a unique on duplicate survey. Will do that now.

Comment (wasade) [inline amgut/db/patches/0029.sql]: thanks!

**Comment by squirrelo** [comment]
Change made, ready to go/

**Pull request merged**